src/colorizer/types.ts

```diff
@@ -164,31 +164,6 @@ export const isTabType = (tab: string): tab is TabType => {
   return Object.values(TabType).includes(tab as TabType);
 };
 
-/**
- * Configuration for the viewer. These are high-level settings
- * that are not specific to a particular dataset.
- */
-export type ViewerConfig = {
-  showTrackPath: boolean;
-  showScaleBar: boolean;
-  showTimestamp: boolean;
-  showLegendDuringExport: boolean;
-  showHeaderDuringExport: boolean;
-  keepRangeBetweenDatasets: boolean;
-  backdropVisible: boolean;
-  /** Brightness, as an integer percentage. */
-  backdropBrightness: number;
-  /** Saturation, as an integer percentage. */
-  backdropSaturation: number;
-  /** Object opacity when backdrop is visible, as an integer percentage. */
-  objectOpacity: number;
-  outOfRangeDrawSettings: DrawSettings;
-  outlierDrawSettings: DrawSettings;
-  outlineColor: Color;
-  openTab: TabType;
-  vectorConfig: VectorConfig;
-};
-
```

Comment:
`Export` was the last place `ViewerConfig` was being used, and I somehow missed it during my cleanup in #596.